méli-mélo/méli-mélo-en.html

```diff
@@ -3,105 +3,157 @@
+
+<fieldset class="gc-chckbxrdio">
+	<legend id="requirements">Minimum requirements for a new feature are:</legend>
+	<ul class="list-unstyled lst-spcd-2">
+		<li class="checkbox">
+			<input type="checkbox" id="req1">
+			<label for="req1">Must be reusable;</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req2">
+			<label for="req2">Must not contain any interference with WET-BOEW &amp; GCWeb, meaning no component override nor conflict. Component extension should be contributed in the <a href="../components/provisional-en.html">provisional</a> or stable component instead;</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req3">
+			<label for="req3">Must be built with accessibility and security in mind. Accessibility conformance remains the responsibility of the publisher when implementing the feature into a web page;</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req4">
+			<label for="req4">Must not impact any content by default on page load unless the feature is explicitly activated through the HTML, either by using a CSS class or data attribute, for example <a href="https://wet-boew.github.io/wet-boew/demos/helloworld/helloworld-en.html">sample WET-BOEW plugin</a>;</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req5">
+			<label for="req5">Must have its name in the right format (see <a href="#feature-name">feature name example below</a>).</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req6">
+			<label for="req6">Must be <a href="#sponsor" rel="help">sponsored<sup aria-hidden="true"><span class="fas fa-info-circle"></span></sup></a> by a department, with an active representative of that department;</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req7">
+			<label for="req7">Must include an <a href="#implementation-plan" rel="help">implementation plan<sup aria-hidden="true"><span class="fas fa-info-circle"></span></sup></a>;</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="req8">
+			<label for="req8">Must have a demo / working example published for each individual sub-features and styles, meaning each JS configurations and CSS classes respectively;</label>
+		</li>
+	</ul>
+</fieldset>
+
+<h3>Get started</h3>
+
+<p>Below are instructions on how to create a new méli-mélo feature.</p>
 
-<div class="well">
-	<p>Features are grouped into <strong>compilations</strong> in order to quickly:</p>
+<div class="alert alert-info">
+	<p><strong>Tip to quickly get started!</strong> Code and/or expose your feature by leveraging <a href="https://github.com/wet-boew/gcweb-jekyll">GCWeb Jekyll theme</a>.</p>
+</div>
+
+<ol class="lst-spcd-2">
+	<li>Create a new feature folder inside the <a href="https://github.com/wet-boew/GCWeb/tree/master/m%C3%A9li-m%C3%A9lo"><code>/méli-mélo</code></a> GCWeb root folder.</li>
```

Comment:
Should we indicate to draft a first prototype by only leveraging GCWeb Jekyll, and then as a second step to proceed with the publication and a second reedition of the méli-mélo feature in GCWeb?

Reply:
Here is the bullet:
`<li>Draft your first contribution prototype by only leveraging GCWeb Jekyll or your environment of choice with the feature code itself being all included within one (1) Javascript file and/or one (1) CSS file.</li>`